### PR TITLE
Nic kendall

### PR DESCRIPTION
**Pull request by NWKendall**
Day 1 - MVP (Stack  & Queue) - SLL not extracted.

### CODE REVIEW

File: stack/stack.py [hunk 1]
```diff
@@ -1,25 +1,209 @@
...
+  # arrays are quicker as have set size
+  # data is contiguous
+  # elements belong to an index
+  # insertion and deletion are more intensive
+  # memory utilization inefficient
+
+
+# class Stack:
+#     def __init__(self):
+#         self.size = 0
```
Comment (israelgonzalezb) [inline stack/stack.py:32]: We should increment and decrement on push and pop, respectively, and hold that counter in this variable. Accessing len() of self.storage multiple times could be an issue as the size of the storage gets larger and larger

Comment (israelgonzalezb) [inline stack/stack.py:32]: We should increment and decrement on push and pop, respectively, and hold that counter in this variable. Accessing len() of self.storage multiple times could be an issue as the size of the storage gets larger and larger

Comment (israelgonzalezb) [inline stack/stack.py:32]: We should increment and decrement on push and pop, respectively, and hold that counter in this variable. Accessing len() of self.storage multiple times could be an issue as the size of the storage gets larger and larger

Comment (NWKendall) [reply stack/stack.py:32]: Thanks, I corrected the __init__ function with latest pull request.

**Pull request merged**